ProcessMaker/processmaker
# QA Observation Bugfix/FOUR-13079: The column layout of the request tables is different from that of Figma.

## How to Test
- Create a new request
- Open Process with request in "Open in Launchpad"
- Check Request Tab columns

## Related Tickets & Packages
https://processmaker.atlassian.net/browse/FOUR-13079

## Code Review Checklist
ci:deploy
ci:next
